Representable/representable
# Editing

definitely not going to get merged in first go...

known problems:
- CSS (and on mobile)
- email sending (probably won't work locally, try on staging)

testing:
so much stuff that it's probably easier just to talk in person... few general things though:

- collectstatic
- makemigrations/migrate
- try out all the general flows and test with drives, no drives, new census blocks, old census blocks (IL/OK), tags (maybe it won't work with tags since I didn't do editing with tags), multiple users collaborating, etc.
- make sure hashing works with everything, also there's a new check-hashing.py that i need to send out, so be on lookout for that
